### DISCUSSION TITLE
Is there plans for native Windows server?

### DISCUSSION THREAD

**Discussion by Drakemoor** [immich-app/immich#6768] [General]
Are there plans for a native Windows server? Or the installation will always have to be with Docker?

**Comment by mmomjian** [comment]
What use case would this provide that can't be accomplished with Docker?

**Comment by Drakemoor** [comment]
I don't understand Docker. It's too complicated for a non-programmer. I don't have time, nor do I want to spend the mental effort to learn it. I want something that is as easy to install as click 1, 2, 3, done, maintains itself with seamless auto-updates, and change settings with a few clicks, not by wading through documentation and deciphering files with strange extensions.

For example: I knew about [AudioBookShelf ](https://github.com/advplyr/audiobookshelf) project for years, but installed it only recently because they released [audiobookshelf-windows](https://github.com/mikiher/audiobookshelf-windows) 

I'm looking for something like that but for photos.

**Comment by mmomjian** [comment]
Given a lot of the dependencies of Immich, including a full Postgres database with extensions, I don't think it is realistically possible to deploy Immich in the way you are looking for. It looks like Audiobookshelf was rewritten in C# to run on Windows.

**Comment by Drakemoor** [comment]
That's unfortunate. 

Anyway, thanks for your answer.
